lib/active_campaign/errors.rb

```diff
@@ -29,12 +29,12 @@ def message
         super
       else
         <<~MESSAGE
-          STATUS: #{response.status}
-          URL: #{env.url}
-          REQUEST HEADERS: #{env.request_headers}
-          RESPONSE_HEADERS: #{env.response_headers}
-          REQUEST_BODY: #{env.request_body}\n\n"
-          RESPONSE_BODY: #{response.body}\n\n"
+          STATUS: #{response.response[:status]}
```

Comment:
why the `response.response`? is that just a case of bad naming?

Reply:
yeah, OP named the variable `response` and I don't want to diverge too much from the original gem implementation to keep simple to merge upstream changes in the future.

Reply:
yea, the original implementation has bugs that doesn't pull data from right source. I had to introduce this change to fix it.